File: northkarana/player.pl
#BeginFile: northkarana\player.pl
#Quest file for North Karana: Paladin message during Necromancer Epic 1.5 (Soulwhisper)

sub EVENT_LOOT {
    if (($looted_id == 14344) && 
        plugin::check_hasitem($client, 26896) && 
        plugin::check_hasitem($client, 11430) && 
        plugin::check_hasitem($client, 22892)) { # All 4 Paladin Heads
        $client->Message(15, "With his last breath, the paladin says, 'You are too late. The last paladin has fled to Natimbi with the staff and is on his way to destroy it!'");
    }
    
    if ($class eq "Ranger" && $looted_id == 62604) {
        if (defined $qglobals{ranger_epic15_pre} && $qglobals{ranger_epic15_pre} ne "8") {
            return 1;
        } else {
            return 0;
        }
    }
}

#EndFile: northkarana\player.pl

File: qey2hh1/player.lua
-- BeginFile: qey2hh1\player.pl
-- Quest file for Adventurer's Stone
-- Quest file for West Karana: Paladin message during Necromancer Epic 1.5 (Soulwhisper)

function event_loot(e)
	if(e.item:GetID() == 26896 and e.self:HasItem(11430) == true and e.self:HasItem(14344) == true and e.self:HasItem(22892) == true) then
		e.self:Message(15, "With his last breath, the paladin says, 'You are too late. The last paladin has fled to Natimbi with the staff and is on his way to destroy it!'");
	end
end

-- EndFile: qey